src/chapters/enums/enums-in-models.rst

```diff
@@ -0,0 +1,16 @@
+Enums in Model Classes
+======================
+
+One application of enum types is to represent categories of objects. We will take this approach in our ``coding-events`` application to categorize events based on their type, such as *conference* or *meetup*.
+
+Enum Types in Models - Video
+----------------------------
+
+.. raw:: html
+
+   <div style="text-align:center;"><iframe width="800" height="450" src="https://www.youtube.com/embed/FOvBYJxGPTQ" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></div>
+
```

Comment:
video looks good! a minor, probably negligible thing is that you see some kind of warning about the th:field on the select element. did you notice that?